### PR TITLE
Add FAQ doc

### LINKED ISSUE #302: Add a Q&A section/page in our documentation (and possible a link in README)
**Is your feature request related to a problem? Please describe.**

I think we have accumulated several commonly-meeted troubles from our end users, in issues, discussions as well as our Dingding talks. It would be helpful if we could provide a Q & A section/page in documentation for user-facing problems as a guidance for newcomers.

### PR DESCRIPTION
**Pull request by lidongze0629**
## Related issue number

Fixes #302

### CODE REVIEW

File: docs/zh/frequently_asked_questions.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+*4. 为什么在使用 `kubectl get pod`命令时，发现了一些多余的pod?*
+
+    对于其中一些失败的pod，只能通过手动命令如 `kubectl delete pod <pod_name>` 进行删除。
+    通常情况下，该问题出现在使用Helm部署GraphScope时，目前，GraphScope依赖一些权限来删除运行时的资源，详细细节可以查看 `Helm Support <https://artifacthub.io/packages/helm/graphscope/graphscope>`_
+
+
+*5. GraphScope是图数据库嘛？*
+
+    GraphScope并不是一个图数据库，其无法提供图上的一些事物性操作。相反，GraphScope使用“immutable”的分布式内存存储以支持快速的图查询和图分析操作。
```
Comment (yecol) [inline docs/zh/frequently_asked_questions.rst:39]: ...事务操作。GraphScope 将图分布式地载入内存作为不可变数据，来支持对该图的查询和分析操作。

File: docs/zh/frequently_asked_questions.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+*7. GraphScope运行过程中常见的卡住原因*
+
+    - 在session拉起阶段，如果当前网络下载镜像过慢，或当前集群资源无法满足pod的拉起时，会造成一定的卡住现象。
+    - 载图阶段，可能会由于数据量过大造成短暂的卡住现象。
+    - 在执行图算法分析阶段，如果当前系统资源紧张，编译构建app时会花费一些时间。
+
+
+*8. 其他问题*
+
+    您可以通过 `issue的方式 <https://github.com/alibaba/GraphScope/issues/new/choose>`_ 或 `在讨论区 <https://github.com/alibaba/GraphScope/discussions>`_ 提出你的问题，同时，您也可以使用 `Slack <http://slack.graphscope.io>`_ 或 `DingTalk <#>`_ 与我们联系。
```
Comment (yecol) [inline docs/zh/frequently_asked_questions.rst:56]: dingtalk link:    https://h5.dingtalk.com/circle/healthCheckin.html?dtaction=os&corpId=ding82073ee2a22b2f86748126f6422b5d02&109d1=d3892&cbdbhh=qwertyuiop

File: README.md [hunk 3]
```diff
@@ -8,9 +8,13 @@
...
 [![Coverage](https://codecov.io/gh/alibaba/GraphScope/branch/main/graph/badge.svg)](https://codecov.io/gh/alibaba/GraphScope)
 [![Docs-en](https://shields.io/badge/Docs-English-blue?logo=Read%20The%20Docs)](https://graphscope.io/docs)
+[![FAQ-en](https://img.shields.io/badge/FAQ-English-lightgrey?logo=Read%20The%20Docs)](https://graphscope.io/docs/frequently_asked_questions.html)
 [![Docs-zh](https://shields.io/badge/Docs-%E4%B8%AD%E6%96%87-blue?logo=Read%20The%20Docs)](https://graphscope.io/docs/zh/)
+[![FAQ-zh](https://img.shields.io/badge/FAQ-%E4%B8%AD%E6%96%87-lightgrey?logo=Read%20The%20Docs)](https://graphscope.io/docs/zh/frequently_asked_questions.html)
 [![README-zh](https://shields.io/badge/README-%E4%B8%AD%E6%96%87-blue)](README-zh.md)
 [![Playground](https://shields.io/badge/JupyterLab-Try%20GraphScope%20Now!-F37626?logo=jupyter)](https://try.graphscope.app)
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/graphscope)](https://artifacthub.io/packages/helm/graphscope/graphscope)
+[![LICENSE](https://img.shields.io/github/license/etcd-io/etcd.svg?style=flat-square)](https://github.com/etcd-io/etcd/blob/main/LICENSE)
```
Comment (yecol) [inline README.md:17]: ???

File: README.md [hunk 4]
```diff
@@ -8,9 +8,13 @@
 [![GraphScope CI](https://github.com/alibaba/GraphScope/workflows/GraphScope%20CI/badge.svg)](https://github.com/alibaba/GraphScope/actions?workflow=GraphScope+CI)
```
Comment (yecol) [inline README.md:8]: maybe sort the badges:
line-1: CI, coverage, try, helm
line-2: docs...

**Review by yecol — CHANGES_REQUESTED** [review]

File: docs/zh/frequently_asked_questions.rst [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+*1. 运行GraphScope系统的最小资源是多少？*
+
+    在通过Python客户端使用GraphScope过程中，需要的最低Python版本是3.6+，最低pip版本是19.0+，GraphScope系统支持单机和分布式的方式部署，单机情况下，要求CPU至少4核，内存至少8G；
+    同时，GraphScope系统支持CentOS7+、Ubuntu18+、以及Kubernetes v1.12.0+
+
+
+*2. GraphScope是否强依赖Kubernetes?*
+
+    GraphScope不强依赖Kubernetes，同时也支持在 `本地环境环境下部署 <https://graphscope.io/docs/deployment.html#deployment-on-local>`_
+    然而，由于Graphscope依赖众多第三方的库和项目，为了节省你的时间，我们建议基于我们提供的镜像构建并运行GraphScope，如果你没有一个运行的Kubernetes集群，可以使用官方提供的 `kind <https://kind.sigs.k8s.io/>`_ 来快速搭建一个集群。
```
Comment (yecol) [inline docs/zh/frequently_asked_questions.rst:16]: 来快速搭建一个集群。->  在本地快速搭建一个虚拟集群。

File: docs/zh/frequently_asked_questions.rst [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    如果您的GraphScope运行在k8s集群上，可以使用 `kubectl describe/logs <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands>`_ 来查看系统的当前状态，同时如果你挂载可本地磁盘，也可在 `/tmp/graphscope/runtime/logs` 目录下查看运行时日志。
+
+
+*4. 为什么在使用 `kubectl get pod`命令时，发现了一些多余的pod?*
+
+    对于其中一些失败的pod，只能通过手动命令如 `kubectl delete pod <pod_name>` 进行删除。
+    通常情况下，该问题出现在使用Helm部署GraphScope时，目前，GraphScope依赖一些权限来删除运行时的资源，详细细节可以查看 `Helm Support <https://artifacthub.io/packages/helm/graphscope/graphscope>`_
+
+
+*5. GraphScope是图数据库嘛？*
```
Comment (sighingnow) [inline docs/zh/frequently_asked_questions.rst:37]: 嘛 -> 吗

File: docs/zh/frequently_asked_questions.rst [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+    GraphScope并不是一个图数据库，其无法提供图上的一些事物性操作。GraphScope 将图分布式地载入内存作为不可变数据，来支持对该图的查询和分析操作。
+
+
+*6. GraphScope在Gremlin上的支持性如何？*
```
Comment (sighingnow) [inline docs/zh/frequently_asked_questions.rst:42]: 支持性 -> 兼容性

File: docs/zh/frequently_asked_questions.rst [hunk 8]
```diff
@@ -0,0 +1,56 @@
+常见问题
+========
+
+下面是用户在使用GraphScope过程中遇到的常见问题，如果你的问题仍无法解决，可通过 `issue的方式 <https://github.com/alibaba/GraphScope/issues/new/choose>`_ 或 `在讨论区 <https://github.com/alibaba/GraphScope/discussions>`_ 与我们联系。
```
Comment (sighingnow) [inline docs/zh/frequently_asked_questions.rst:4]: 下面是用户在使用GraphScope过程中遇到的常见问题，如果你的问题仍无法解决 -> 我们收集了用户使用GraphScope过程中的常见问题，如果在下面的列表中仍然无法找到您遇到的问题的答案

File: docs/frequently_asked_questions.rst [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    - Ubuntu18+
+
+    For distributed depolyment, a cluster managed by Kubernetes is required. GraphScope has tested on
+    k8s **version >= v1.12.0+**.
+
+
+*2. Is Kubernetes an enssential to run GraphScope?*
+
+    No. GraphScope supports `build and run on local <https://graphscope.io/docs/deployment.html#deployment-on-local>`_ in a single machine. 
+    However, GraphScope depends on many third-party libraries and projects. To make our life easier, we suggest you build and run GraphScope on k8s with our provided dev-image and release-image. If you don't have an k8s managed cluster, you may use tools like `kind <https://kind.sigs.k8s.io/>`_ to setup a local cluster to use the images.
```
Comment (sighingnow) [inline docs/frequently_asked_questions.rst:21]: ”To make our life easier“ -> To make the setup process easier

Comment (sighingnow) [reply docs/frequently_asked_questions.rst:21]: an k8s managed cluster -> a kubernetes cluster.

Comment (sighingnow) [reply docs/frequently_asked_questions.rst:21]: to use the images -> to try graphscope.

File: docs/frequently_asked_questions.rst [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+
+*3. How to debug or get detailed information when run GraphScope?*
+
+    By default, GraphScope is mostly running in a silent mode following the style of Python.
```
Comment (sighingnow) [inline docs/frequently_asked_questions.rst:26]: mostly -> usually

Comment (sighingnow) [reply docs/frequently_asked_questions.rst:26]: the style of Python -> the convention of Python applications.

File: docs/frequently_asked_questions.rst [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+
+*6. What's the compatibility of Gremlin in GraphScope?*
+
+    GraphScope supports most querying operators in Gremlin. You may check the compatibility in this `link <https://graphscope.io/docs/interactive_engine.html#unsupported-features>`_.
+
+
+*7. The system seems get stuck, what are the possible reasons?*
+
+    If GraphScope seems get stuck, the possible reasons:
```
Comment (sighingnow) [inline docs/frequently_asked_questions.rst:55]: Seem to get...

the possible cause might be

File: docs/frequently_asked_questions.rst [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+
+    - In the session launching stage, the most cases are waiting for pods ready. The time consuming may be caused by a poor network connection during pulling image, or caused by the resources cannot meet the need to launch a session.
+    - In the graph loading stage, it is time consuming to load and build a large graph.
+    - In the analysis stage with a UDF/built-in algorithm, it takes time to compile/distribute the algorithm over the loaded graph.
```
Comment (sighingnow) [inline docs/frequently_asked_questions.rst:59]: When running a user-defined or built-in analytical algorithm

**Review by sighingnow — APPROVED** [review]
LGTM, with some minor comments.

**Review by yecol — APPROVED** [review]

**Review by yecol — APPROVED** [review]

**Review by yecol — APPROVED** [review]

**Pull request merged**